### DISCUSSION TITLE
self-hosting campus-imaotai v1.0.12

### DISCUSSION THREAD

**Discussion by woodwm** [oddfar/campus-imaotai#234] [Q&A]
In the campus-imaotao v1.0.12, there are two packages "[campus-imaotai-web](https://github.com/users/oddfar/packages/container/package/campus-imaotai-web)" and "[campus-imaotai-api](https://github.com/users/oddfar/packages/container/package/campus-imaotai-api)". No v1.0.12 on docker hub https://hub.docker.com/r/oddfar/campus-imaotai/tags.

How to self-hosting campus-imaotai v1.0.12?

**Comment by oddfar** [comment]
不打算推docker hub了，可参考 `GitHub Action构建并推送DockerHub` 教程
改写本项目的 [api.yml](https://github.com/oddfar/campus-imaotai/blob/master/.github/workflows/api.yml) 和 [web.yml](https://github.com/oddfar/campus-imaotai/blob/master/.github/workflows/web.yml) 自行托管
campus-imaotai-web 是前端镜像、campus-imaotai-api是后端镜像
